src/v2/view-builder/internals/BaseForm.ts

```diff
@@ -104,7 +104,7 @@ export default Form.extend({
 
     const header = this.$el.find('[data-se="o-form-head"]');
     const identifierHTMLString = hbs('<div class="identifier-container">\
-        <span class="identifier no-translate" data-se="identifier" title={{identifier}}>{{identifier}}</span>\
+        <span class="identifier no-translate" data-se="identifier" title="{{identifier}}">{{identifier}}</span>\
```

Comment:
LGTM. Handlebars doesn't auto-escape (like React)? I thought there was something with `{{evil}}` vs `{{{evil}}}`

Reply:
It does escape with the double curly braces, which we were doing and it worked fine with the identifier being passed into the span content, but it wasn't working well with the attribute, without it being in quotes.
The triple curly braces syntax is the evil one, that does not escape :)